src/ui/components/RecommendedBadge/index.js

```diff
@@ -0,0 +1,41 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import Icon from 'ui/components/Icon';
+import type { I18nType } from 'core/types/i18n';
+
+import './styles.scss';
+
+type Props = {};
+
+type InternalProps = {|
+  i18n: I18nType,
+|};
+
+export const RecommendedBadgeBase = (props: InternalProps) => {
+  const { i18n } = props;
+
+  const label = i18n.gettext('Recommended');
+  return (
+    <div className="RecommendedBadge">
+      <a
+        className="RecommendedBadge-link"
+        href="https://support.mozilla.org/"
```

Comment:
Do we need UTM params on this link? /cc @jvillalobos

Reply:
Good question. I assume that will all be revealed when we get the final URL, but I should try to remember to check for UTM params.

Reply:
Already done! :-) https://github.com/mozilla/addons-frontend/issues/7961